dev-util/phpstorm/phpstorm-2017.3.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils vcs-snapshot
```

Comment:
Does upstream use unpredictable path name so that you need to use vcs-snapshot?

Reply:
Yes, see https://github.com/gentoo/gentoo/pull/216#issuecomment-151558278 :wink: